src/Integration.UnitTests/Binding/SolutionBindingOpearation.WriterTests.cs

```diff
@@ -0,0 +1,102 @@
+//-----------------------------------------------------------------------
+// <copyright file="SolutionBindingOpearation.WriterTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.VisualStudio.CodeAnalysis.RuleSets;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.Integration.Binding;
+using System;
+using System.Globalization;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests.Binding
+{
+    public partial class SolutionBindingOpearationTests
```

Comment:
It's a rename rather than a new file. This was the tests file for the solution ruleset writer.